Index: ChromeDriver.wiki
===================================================================
--- ChromeDriver.wiki   (revision 16978)
+++ ChromeDriver.wiki   (working copy)
@@ -168,6 +168,7 @@
   # Typing does not work with rich-text enabled documents
   # Cannot specify a custom profile
   # [http://selenium.googlecode.com/svn/trunk/docs/api/java/org/openqa/selenium/remote/html5/package-summary.html HTML 5 API] not implemented
+  # If running Selenium as a Windows Service, be sure to use the alternate 'standalone' chrome install.  This will install chrome for all users, not just the current user.  The Selenium service will try and start Chrome as the SYSTEM user and will not be able to find the binary, as noted in [http://code.google.com/p/chromedriver/issues/detail?id=46 Issue #46].  The alternate install is an easy way to correct that problem.  The alternate install can be [http://www.google.com/chrome/eula.html?system=true&standalone=1 found here].
 
 === Think you've found a bug? ===
